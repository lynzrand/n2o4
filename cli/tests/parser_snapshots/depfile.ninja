# From: Ninja doc, "depfile"
rule cc
  depfile = $out.d
  command = gcc -MD -MF $out.d [other gcc flags here]
