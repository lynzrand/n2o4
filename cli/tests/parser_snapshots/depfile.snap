Ok(
    NinjaFile {
        global_scope: {},
        rules: {
            "cc": Rule {
                vars: {
                    "depfile": Expandable(
                        [
                            Var(
                                "out",
                            ),
                            Regular(
                                ".d",
                            ),
                        ],
                    ),
                    "command": Expandable(
                        [
                            Regular(
                                "gcc -MD -MF ",
                            ),
                            Var(
                                "out",
                            ),
                            Regular(
                                ".d [other gcc flags here]",
                            ),
                        ],
                    ),
                },
            },
        },
        builds: [],
        phony: {},
        defaults: [],
    },
)
